### PR TITLE
BrightcomSSP bid adapter: add new adapter

### PR DESCRIPTION
**Pull request by prBigBrother**
## Type of change

- [X] New bidder adapter   

## Description of change
Adding new BrightcomSSP adapter 

- contact email: alexandruc@brightcom.com
- test parameters for validating bids:
```
{
      bidder: 'bcmssp',
      params: {
        publisherId: 2141020
      }
}
```

### CODE REVIEW

**Comment by prBigBrother** [comment]
prebid.github.io pull request with documentation page for new bidder:

https://github.com/prebid/prebid.github.io/pull/4267

File: modules/brightcomSSPBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+}
+
+function isBidRequestValid(bid) {
+  if (bid.bidder !== BIDDER_CODE || typeof bid.params === 'undefined') {
+    return false;
+  }
+
+  if (typeof bid.params.publisherId === 'undefined') {
+    return false;
+  }
```
Comment (mmoschovas) [inline modules/brightcomSSPBidAdapter.js:132]: Can this be joined as one condition? 

```
if (bid.bidder !== BIDDER_CODE || !bid.params || !bid.params.publisherId) {
    return false;
}
```

File: modules/brightcomSSPBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+  isBidRequestValid,
+  buildRequests,
+  interpretResponse,
+  onBidderError,
+  onTimeout,
+  onBidWon,
+  getUserSyncs,
+};
```
Comment (mmoschovas) [inline modules/brightcomSSPBidAdapter.js:33]: Just want to make sure that the absence of gvlid is intentional but was meant to be added to the other bidder file?

File: modules/brightcomSSPBidAdapter.js (hunk 1 above)
Comment (prBigBrother) [reply modules/brightcomSSPBidAdapter.js:129]: Thanks for noticing, updated!

File: modules/brightcomSSPBidAdapter.js (hunk 2 above)
Comment (prBigBrother) [reply modules/brightcomSSPBidAdapter.js:34]: It's a mistake, I added the gvlid in the spec object. Thank you!

**Review by mmoschovas — APPROVED** [review]

**Comment by mmoschovas** [comment]
@ChrisHuie do you know what happened with the CircleCI tests here? I cant seem to rerun them

**Comment by prBigBrother** [comment]
Hi @mmoschovas and @ChrisHuie!

I hope all is good on your side. I'd like to ask for an update on our pull-request, please. We're excited to move forward!

Thank you,
Alex

**Comment by ChrisHuie** [comment]
Looking at this. Will figure out and get merged this week.

**Pull request merged**